Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM debian:stretch-backports
+
+LABEL authors="@bnmnetp,@vsoch,@yarikoptic"
+
+# docker build -t runstone/server .
+# TODO: convert generation to neurodocker call after all is cool
+
+# Define some ARGs which could be passed into while building
+# TODO: in reality there some hardcoding already probably present
+#       in the entrypoint.sh script.
+ARG WEB2PY_PATH=/srv/web2py
+ARG WEB2PY_APPS_PATH=${WEB2PY_PATH}/applications
+ARG WEB2PY_PORT=8080
+
+# And export some as env vars so they could be available at run time
+ENV WEB2PY_PATH=${WEB2PY_PATH}
+ENV RUNESTONE_PATH=${WEB2PY_APPS_PATH}/runestone
+ENV BOOKS_PATH=${RUNESTONE_PATH}/books
+ENV WEB2PY_VERSION=2.18.4
+
+# Expose that port on the network
+EXPOSE ${WEB2PY_PORT}
+
+# To prevent interactive debconf during installations
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Components from requirements.txt which are available in Debian
+# Missing ones:
+#  runestone -- is the RunestoneComponents, https://pypi.org/project/runestone/, may be install from Git?
+#  paver -- too old in Debian  filed bug report
+#  selenium -- also a bit too old (2.53.2+dfsg1-1)
+#  sphinxcontrib-paverutils -- N/A
+#  sphinx -- we need stretch-backports
+#  pytz ... ?
+# A few missing ones
+#  rsync is needed when deploying a built book
+#  vim - just for pleasure of being able to do any changes right within
+#  wget - just in case
+RUN apt-get update && \
+    apt-get install -y eatmydata && \
+    eatmydata apt-get update && echo "count 1" && \
```

Comment:
BTW, that "count 1" could probably be stripped -- just used it whenever wanted to force redoing this particular layer I guess

Reply:
Ah okay, I figured you were channeling your inner Count!

1... 2.... 3.... hahahaaaaa !